run_update.py

```diff
@@ -44,6 +43,7 @@
 # TODO: use a Meetup client library with pagination
 MEETUP_API_URL = "https://api.meetup.com/2/events?status=past,upcoming&format=json&group_urlname={group_urlname}&key={key}&desc=true&page=200"
 MEETUP_COUNT_API_URL = "https://api.meetup.com/2/groups?group_urlname={group_urlname}&key={key}"
+GITHUB_USER_API_URL = 'https://api.github.com/users/{username}'
```

Comment:
I'm a little confused by the different use of `/user` and `/orgs` below. As long as you are clear on which to use when, then 👍

Reply:
It appears that using `/user/:id` works in both cases - whether the profile is a single user _or_ an organization - and at least one brigade has a single-user profile. But the general case for an organization. Good flag.